xbmc/pvr/dialogs/GUIDialogPVRRadioRDSInfo.cpp

```diff
@@ -0,0 +1,416 @@
+/*
+ *      Copyright (C) 2012-2015 Team KODI
+ *      http://kodi.tv
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with KODI; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "Application.h"
+#include "FileItem.h"
+#include "GUIDialogPVRRadioRDSInfo.h"
+#include "GUIUserMessages.h"
+#include "guilib/GUIWindowManager.h"
+#include "guilib/GUISpinControl.h"
+#include "guilib/GUIMessage.h"
+#include "guilib/GUITextBox.h"
+#include "guilib/LocalizeStrings.h"
+#include "messaging/ApplicationMessenger.h"
+#include "pvr/channels/PVRRadioRDSInfoTag.h"
+
+using namespace PVR;
+
+#define CONTROL_BTN_OK    10
+#define SPIN_CONTROL_INFO 21
+#define TEXT_INFO         22
+#define CONTROL_NEXT_PAGE 60
+#define CONTROL_INFO_LIST 70
```

Comment:
Hi @phil65 about your question with "Radio text info list" skin group, on skin was a mistake from me during testing and the id was temporary removed. Is normally 70 and becomes set visible on the line 108 if availabe.